buildSrc/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
-    `kotlin-dsl`
+    // workaround https://github.com/gradle/gradle/issues/16345
```

Comment:
btw - the issue is from the February... 
Very strange

Reply:
The issue is from February 2021. it's long live issue.
I think that a fix is possible only in Gradle 8.
But comment about `Kotlin 1.7.0` is from June 9th (release date :))